Library/Homebrew/download_strategy.rb

```diff
@@ -410,7 +410,7 @@ def fetch(timeout: nil)
         use_cached_location = cached_location.exist?
         use_cached_location = false if version.respond_to?(:latest?) && version.latest?
 
-        resolved_url, _, last_modified, _, is_redirection = begin
+        resolved_url, _, last_modified, file_size, is_redirection = begin
           resolve_url_basename_time_file_size(url, timeout: Utils::Timer.remaining!(end_time))
```

Comment:
Maybe not in this PR, but both the name and return type strongly suggest using a URLMetadata (or such) struct here instead.

Reply:
Maybe someday 😅